src/data/rat/default.lean

```diff
@@ -0,0 +1,12 @@
+/-
```

Comment:
@robertylewis Do we want docs for `default` files as well?

Reply:
I mean, it can't hurt, but I don't think it's so important.

Reply:
I mean, it can't hurt, but I don't think it's so important.

Reply:
Ok, I'm inclined to say that these files are "content-free" so they don't need any docs. But I don't really care.